libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/MapboxEnablableButtonParams.kt

```diff
@@ -0,0 +1,44 @@
+package com.mapbox.navigation.dropin
+
+/**
+ * Params of a button that can be enabled.
+ *
+ * @param enabled true if it should be enabled, false otherwise
+ * @param buttonParams [MapboxExtendableButtonParams] of the button
+ */
+class MapboxEnablableButtonParams(
```

Comment:
I'm open for other suggestions on the API.

Reply:
Based on your comment I get the impression you don't like this solution.

Here's my alternative:
- split `ActionButtonBinder` into smaller binders - one for each button
- get rid of all `MapboxExtendableButtonParams` and instead expand `ViewBinderCustomization` to allow injection of a custom binder for each button
- introduce factory method for each button binder eg. `ViewBinderCustomization.compassButtonBinder()`
- allow enabling/disabling of a compass by setting its binder:

```kotlin
if (enabled) {
    binding.navigationView.customizeViewBinders {
        compassButtonBinder = compassButtonBinder() // use our default CompassButtonBinder or provide your own
    }
} else {
    binding.navigationView.customizeViewBinders {
        compassButtonBinder = UIBinder.USE_DEFAULT
    }
}
```
@abhishek1508 WDYT?

Reply:
Idk if we should change the overall approach. Although it would allow to show/hide some specific buttons. However, it can be done in a simpler way: we just change all the existing buttons params from `MapboxExtendableButtonParams` to `MapboxEnablableButtonParams`. Or we can just add a `enabled` property to `MapboxExtendableButtonParams`.

I don't like the "enablable" word. :)
Also I'm wondering if I should:
1) Extend `MapboxExtendableButtonParams` instead of injecting itl
2) Use builder instead of public constructor.

Otherwise I don't have anything against the current approach.

Reply:
https://github.com/mapbox/mapbox-navigation-android/pull/6395#discussion_r981402128 would be a good solution as it would keep this new button and others in sync with the overall architecture being used for other components. However this is outside the scope of this PR. I have cut a ticket tracking the request in NAVAND-711.
For this PR let us move forward with just adding `enabled` flag to `MapboxExtendableButtonParams`. 
@dzinad